src/Common/Telemetry/Telemetry.php

```diff
@@ -462,13 +467,12 @@ public static function get_tec_telemetry_slugs() {
 	 * @return void
 	 */
 	public function register_tec_telemetry_plugins( $opted = NULL ) {
-		$new_opted = $opted;
 		// Let's reduce the amount this triggers.
-		if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
```

Comment:
This got moved to `boot()`

Reply:
Because the add-ons call `register_tec_telemetry_plugins()` (for now), I think we should keep this here.